src/components/app/data/utils.js

```diff
@@ -373,11 +372,9 @@ export const transformCourseEnrollment = (rawCourseEnrollment) => {
   courseEnrollment.hasEmailsEnabled = courseEnrollment.emailsEnabled;
   courseEnrollment.notifications = courseEnrollment.dueDates;
   courseEnrollment.canUnenroll = canUnenrollCourseEnrollment(courseEnrollment);
-  courseEnrollment.isCourseAssigned = false;
```

Comment:
[inform] This particular `isCourseAssigned` property on realized enrollments doesn't appear to actually be consumed anywhere.